src/deserializer.mts

```diff
@@ -44,11 +44,14 @@ const printAttr = ({
   hasDefaultValue,
   columnName,
   kind,
+  dbName,
+  name,
 }: Field) =>
   (isId ? '@id' : '') +
   (isUnique ? ' @unique' : '') +
   (isUpdatedAt ? ' @updatedAt' : '') +
   (columnName ? ` @map(${JSON.stringify(columnName)})` : '') +
+  (!columnName && dbName ? ` @map(${JSON.stringify(name)})` : '') +
   (hasDefaultValue ? ' ' + printDefault(kind, def) : '');
```

Comment:
I'm not sure if this condition is true, but I just assumed you can't have `@map` defined twice so I wrote it this way